Corzebank.Extensions/ExceptionMiddlewareExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Corzbank.Data.Entities.Models;
+using Corzbank.Services.Interfaces;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Corzbank.Extensions
+{
+    public static class ExceptionMiddlewareExtension
+    {
+        public static void ConfigureCustomExceptionMiddleware(this IApplicationBuilder app)
```

Comment:
rename to ConfigureExceptionHandlingMiddleware

Reply:
resolved